lightly_studio_view/src/lib/hooks/useImagesInfinite/useImagesInfinite.ts

```diff
@@ -118,10 +120,15 @@ const buildRequestBody = (params: ImagesInfiniteParams, pageParam: number): Read
             ...baseBody,
             filters: {
                 ...baseBody.filters,
-                annotation_label_ids: params.filters.annotation_label_ids?.length
-                    ? params.filters.annotation_label_ids
-                    : undefined,
-                tag_ids: params.filters.tag_ids?.length ? params.filters.tag_ids : undefined,
+                sample_filter: {
```

Comment:
This overwrite the existing `sample_filter` which includes the `metadata_filters`.